### ISSUE TITLE
`next` and `canary` releases publishes all packages in Lerna independent mode

### ISSUE THREAD

**Issue by glambert** [intuit/auto#815]
**Describe the bug**

Both `next` and `canary` release pipelines while using `auto shipit` publishes **all packages** even the ones that are unchanged in a lerna monorepo in independent mode.

**To Reproduce**

- See `next` example here: https://github.com/glambert/auto-lerna-independent-poc/pull/9
- See `canary` example here: https://github.com/glambert/auto-lerna-independent-poc/pull/6 (and the GitHub Action workflow: https://github.com/glambert/auto-lerna-independent-poc/pull/6/checks?sha=1cfc5a6100b09cf06f2291221833457c027014aa)

**Expected behavior**

Only create `next` and `canary` releases on changed packages.

**Screenshots**

**Desktop (please complete the following information):**

- OS: all
- Browser: all
- Version: 8

**Additional context**

Would it be possible to use [`lerna changed`](https://github.com/lerna/lerna/tree/master/commands/changed) and only run `--force-publish` on those packages for `next` and `canary` releases?

Kudos for this amazing library again, let me know if I can help with anything :+1:

**Comment by ericclemmons** [comment]
I'm also interested in this answer for https://github.com/aws-amplify/amplify-js/.  

@glambert Your reproduce links don't work :(

**Comment by hipstersmoothie** [comment]
This is actually intentional for the fixed workflow. We can change that though if it makes sense for independent mode.

Relevant lines: 

https://github.com/intuit/auto/blob/master/plugins/npm/src/index.ts#L702
https://github.com/intuit/auto/blob/master/plugins/npm/src/index.ts#L579

Also to note the Github Release Page in `independent` mode could use some workhttps://github.com/intuit/auto/issues/656

But if you're looking to use this for `amplify` I would be happy to address this 😄

**Comment by glambert** [comment]
> I'm also interested in this answer for https://github.com/aws-amplify/amplify-js/.
> 
> @glambert Your reproduce links don't work :(

Yeah I know, I had to switch to a private repo to some more testing. If needed I'll re-open a repo to show the reduced test case.

**Comment by glambert** [comment]
FYI, this is what I'm getting with the latest version (`9.8.0`) for canary releases (PR messages do work now, but don't show the version):

```
ℹ  info      Using command: 'pr-body'
ℹ  info      Using PR body identifier: <!-- GITHUB_RELEASE PR BODY: canary-version -->
ℹ  info      Getting previous pr body on: 12
ℹ  info      Creating new pr body
ℹ  info      Successfully updated body of PR #12
✔  success   Updated body on PR #12
✔  success   Published canary version: Canary Versions
```

**Comment by adierkens** [comment]
:rocket: Issue was released in `v9.15.3` :rocket:
